docs/developer-guide/etendo-classic/concepts/format-xml.md

```diff
@@ -0,0 +1,68 @@
+---
+tags:
+  - Configuration
+  - Numeric Formatting
+  - XML Configuration
+  - Application Dictionary
+  - Format Mask
+---
+
+##Overview
+
+`Format.xml` is a configuration file for Etendo installation that allows configuring the format output for numeric values. It is used by the different **numeric references** in Application Dictionary, but also can be used in **manual** code. By default, Etendo ships a `Format.xml.template` that can be copied as it is without any modification.
+
+!!!info
+    If you are building Etendo from sources, the setup-properties binary makes a copy of it for you.
+
+## Format.xml example
+
+```xml
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <!-- license -->
+    <Formats>
+    <!-- other formats -->
+       <Number name="euroEdition"
+           decimal="." grouping="," formatOutput="#0.00" formatInternal="#0.00" />
+    <!-- other formats -->
+    </Formats>
+```
+
+### Attributes
+
+  * **name**: Name of the format, used to identify it.
+  * **decimal**: Symbol (character) to be used as decimal separator.
+  * **grouping**: Symbol (character) to be used as grouping separator (used in thousands).
+  * `formatOutput`: Format mask used to mask and print numeric inputs. It must be `DecimalFormat` output format type: For more information, see [DecimalFormat](https://docs.oracle.com/javase/tutorial/java/data/numberformat.html){target="\_blank"} class.
+  * `formatInternal`: Used internally by `XmlEngine`.
+
+## Application dictionary - format name mapping
+
+| AD Reference     | `Output format`     |
+|------------------|---------------------|
+| Decimal, Amount  | `euroEdition`       |
+| Quantity         | `qtyEdition`        |
+| Price            | `priceEdition`      |
+| Integer          | `integerEdition`    |
+| Number           | `generalQtyEdition` |
+| Others numeric   | `generalQtyEdition` |
+
+## Important Notes
+
+  * You can define a decimal and grouping separator per format type, but the `qtyEdition` format is the one that defines them. Makes no sense to define different decimal and grouping per format type.
+  * Rounding numeric values is also affected by the mask of the input. If you want more precision, change the output format attribute.
+  * If you do not already have a `Format.xml` in the WEB-INF directory of your live Etendo installation and want to experiment with various settings, then copy `$openbravo_source/config/Format.xml.template` to `$live_openbravo_installation/WEB-INF/Format.xml`, edit it and restart Tomcat.
+
+!!!note
+    Ultimately, copy `$openbravo_source/config/Format.xml.template` as `Format.xml` and `ant compile -Dtab=xx -Dtr=no` because this way the changes will be permament and will not be lost upon the next rebuild.
```

Comment:
```suggestion
    Ultimately, copy `config/Format.xml.template` as `Format.xml` and `ant compile -Dtab=xx -Dtr=no` because this way the changes will be permament and will not be lost upon the next rebuild.
```